drf_openapi/entities.py

```diff
@@ -301,12 +302,14 @@ def get_serializer_fields(self, path, method, view, version=None):
                 continue
 
             required = field.required and method != 'PATCH'
+            # if the attribute ('help_text') of this field is a lazy translation object, force it to generate a string
+            description = str(field.help_text) if isinstance(field.help_text, Promise) else field.help_text
```

Comment:
I wonder if we can just do `str(field.help_text)` regardless of the condition?

Reply:
We can do that, but I have a concern that If the data type of `field.help_text` is `None`, the description will become `"None"` instead of empty string. It might mislead/confuse users while viewing the schema.

![image](https://user-images.githubusercontent.com/12512422/31271886-3c5a775c-aa89-11e7-9931-7a151476dca3.png)

Reply:
Good point